From: Jun Yang <jun.yang@suse.com>
Date: Wed, 16 Feb 2022 10:30:43 +0800
Subject: [PATCH] spnic: fix compile issue of spnic_netdev_ops.c
Patch-mainline: Never, fixes out-of-tree driver
References: bsn#92
Modified-by-SEL: N/A

mainline commit cc69837fcaf467426ca19e5790085c26146a2300
(net: don't include ethtool.h from netdevice.h)
drop the linux/ethtool.h from netdevice.h,
so need include ethool.h in the driver.

fix build error in spnic driver follow this commit.

Signed-off-by: Jun Yang <jun.yang@suse.com>
---
 drivers/net/ethernet/ramaxel/spnic/spnic_netdev_ops.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/ramaxel/spnic/spnic_netdev_ops.c b/drivers/net/ethernet/ramaxel/spnic/spnic_netdev_ops.c
index a4f668682f37..421d7c64adbd 100644
--- a/drivers/net/ethernet/ramaxel/spnic/spnic_netdev_ops.c
+++ b/drivers/net/ethernet/ramaxel/spnic/spnic_netdev_ops.c
@@ -10,6 +10,7 @@
 #include <linux/etherdevice.h>
 #include <linux/netdevice.h>
 #include <linux/netlink.h>
+#include <linux/ethtool.h>
 #include <linux/debugfs.h>
 #include <linux/ip.h>
 #include <linux/bpf.h>
-- 
2.34.1

